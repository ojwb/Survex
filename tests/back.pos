( Easting, Northing, Altitude )
(    0.00,     0.00,     0.00 ) 1
(    0.00,     0.87,    -0.50 ) 2
(    0.00,     0.87,    -0.50 ) 3
(    0.00,     0.87,    -0.50 ) 4
(    0.00,     0.87,    -0.50 ) 5
(    0.00,     0.87,    -0.50 ) a
(    0.00,     0.87,    -0.50 ) b
(    0.00,     0.87,    -0.50 ) c
(    0.00,     0.87,    -0.50 ) d
(    0.00,     0.87,    -0.50 ) e
(   -1.00,    -1.00,    -1.00 ) min
(    9.00,    12.00,    36.00 ) max
(    0.00,    10.00,     0.00 ) r
(    0.00,    10.00,     0.00 ) x
(    0.00,    10.00,     0.00 ) y
(    0.00,    10.00,     0.00 ) z
