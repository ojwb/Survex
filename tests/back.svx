; pos=yes warn=2
*data normal from to tape compass clino backcompass backclino
*fix 1 0 0 0
1 2 1.0 000 -30  -   -
1 3 1.0  -   -  180  30
1 4 1.0 000  -   -   30
1 5 1.0  -  -30 180  -
1 A 1.0 000 -30 180  30
1 B 1.0  -  -30 180  30
1 C 1.0 000  -  180  30
1 D 1.0 000 -30  -   30
1 E 1.0 000 -30 180  -
*data normal from to tape compass backcompass clino backclino
*units compass backcompass clino backclino grads
1 X 10 000.5 199.5  00  -
1 Y 10 000  -   02  02
1 Z 10 000  -  -01.25 -01.25
; Regression test for bug fixed in 1.2.25.  In versions prior to this, this
; would incorrectly warn about the difference between COMPASS and BACKCOMPASS,
; as we were using 0 for the BACKCOMPASS s.d. which leads to a threshold of
; about 71% of what it should be:
1 R 10 001.178 198.822 -1.178 -1.178

*data cartesian from to dx dy dz
; Work around variation in which stations are picked for min and max:
1 min -1 -1 -1
1 max 9 12 36
