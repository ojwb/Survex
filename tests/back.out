./back.svx:16:21: warning: CLINO reading and BACKCLINO reading disagree by 4grd
 1 Y 10 000  -   02  02
                     ^~
./back.svx:17:23: warning: CLINO reading and BACKCLINO reading disagree by 2.5grd
 1 Z 10 000  -  -01.25 -01.25
                       ^~~~~~

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 15 survey stations, joined by 14 legs.
There is 1 loop.
Survey has 2 connected components.
Total length of survey legs =   88.00m (  88.00m adjusted)
Total plan length of survey legs =   62.79m
Total vertical length of survey legs =   40.50m
Vertical range = 36.50m (from max at 36.00m to e at -0.50m)
North-South range = 12.00m (from max at 12.00m to 1 at 0.00m)
East-West range = 9.00m (from max at 9.00m to z at 0.00m)
  14 1-nodes.
   1 14-node.
There were 2 warning(s).
